corehq/apps/api/tests/user_resources.py

```diff
@@ -452,6 +479,41 @@ def test_update(self):
         self.assertEqual(modified.first_name, "Joe")
         self.assertEqual(modified.last_name, "Admin")
         self.assertEqual(modified.email, "admin@example.com")
+        self.assertEqual(modified.phone_numbers, ["9999999999", "9899999999"])
+        self.assertEqual(modified.default_phone_number, "9999999999")
+
+        # role is ignored
+        self.assertEqual(modified.get_role(self.domain.name), None)
+
+        # is_admin, permissions are simply set on user but have no visible effect
```

Comment:
should you note that these are likely bugs?

Reply:
You mean in the comment here? I am thinking this would likely get picked soon through support and will get updated accordingly. But happy to update the comment here if that makes sense.
Looks more like an oversight, rather than a bug. I don't even remember if we ever had permissions on a user record ever, so I don't even understand why this was done at the first place.

Reply:
yea, just an update to the comment

Reply:
Reported here, https://dimagi-dev.atlassian.net/browse/SUPPORT-10484

Reply:
Also related, https://dimagi-dev.atlassian.net/browse/SAAS-12603